pycheribuild/projects/cross/gkermit.py

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (c) 2021 Microsoft
```

Comment:
2022?
And surely it's more than just Microsoft? (Microsoft Corporation?)

Reply:
Time keeps on slipping, doesn't it.  And yea, probably "Microsoft Corporation" though we don't appear to be perfectly consistent.